CHANGES.rst

```diff
@@ -6,7 +6,9 @@ Other
 
 - Add helper functions to aid in migration of ASDF-in-FITS
   uses from asdf to this package [#114]
-
+- Move JP-3015 ``jwst.datamodels`` updates from the ``jwst`` package into this
```

Comment:
The entry should describe the actual changes that were made to the code modules.